arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-kenya.dts

```diff
@@ -170,6 +171,34 @@
 		compatible = "microchip,24lc256","atmel,24c256";
 		reg = <0x50>;
 	};
+
+	i2cswitch@70 {
+		compatible = "nxp,pca9546";
+		reg = <0x70>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		channel_0_2: i2c@2 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <2>;
+			pvddcrcpu0@61 {
+				//PVDDCR_CPU0_VDDIO_MEM_S3_P0_CONTROLLER
+				compatible = "isil,isl68137";
+				reg = <0x61>;
+			};
+			pvddcrcpu1@62 {
+				//PVDDCR_CPU1_PVDDCR_SOC_P0_CONTROLLER
+				compatible = "isil,isl68137";
```

Comment:
Kenya VR at 0x62 is RAA229639
does this driver work fir it?

Reply:
We did not have a matching driver for RAA parts in the linux kernel.
However, ISL driver from Renesas is working with the RAA parts. So added ISL as compatible driver for RAA parts.